scripts/App.js

```diff
@@ -57,11 +57,11 @@ history.listen((location) => {
 })
 
 function App() {
-  const [formattedCount, setFormattedCount] = useState('0')
+  const [formattedCount, setFormattedCount] = useState('Over 4.200')
```

Comment:
Change back to '0' & add
```py
useEffect(() => {
  if (formattedCount === '0') {
    const newFormattedCount = new Intl.NumberFormat().format(4200)
    setFormattedCount(newFormattedCount)
  }
}, [formattedCount])
```

At line 66 so that on mount it formats the number based upon he users System, see both lines [84](https://github.com/BlobEmoji/blobs.gg/blob/feature/blobs-v2/scripts/App.js#L84) and [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat)